src/tools/__tests__/autosave.test.js

```diff
@@ -0,0 +1,55 @@
+import { updateAutosave, getAutosaveState } from '../autosave'
+import { stateFromJsmd } from '../jsmd-tools'
+import { newNotebook, newCell } from '../../editor-state-prototypes'
+
+describe('updateAutosave', () => {
+  function setUp() {
```

Comment:
`beforeEach` is a Jest function that does this by default, like `beforeEach(()=>{something})`.

Reply:
https://jestjs.io/docs/en/setup-teardown